backend/api/Controllers/EchoMissionController.cs

```diff
@@ -95,4 +95,71 @@ public async Task<ActionResult<EchoMission>> GetEchoMission([FromRoute] int miss
             return new StatusCodeResult(StatusCodes.Status500InternalServerError);
         }
     }
+    [HttpGet]
+    [Route("installation/{installationCode}")]
+    [ProducesResponseType(typeof(EchoMission), StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status400BadRequest)]
+    [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+    [ProducesResponseType(StatusCodes.Status403Forbidden)]
+    [ProducesResponseType(StatusCodes.Status404NotFound)]
+    [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+    [ProducesResponseType(StatusCodes.Status502BadGateway)]
+    public async Task<ActionResult<EchoMission>> GetEchoMissionsFromInstallation([FromRoute] string installationCode)
+    {
+        try
+        {
+            var mission = await _echoService.GetMissionsByInstallation(installationCode);
+            return Ok(mission);
+        }
+        catch (HttpRequestException e)
+        {
+            if (e.StatusCode.HasValue && (int)e.StatusCode.Value == 404)
+            {
+                _logger.LogWarning("Could not find echo mission from installation with installationCode={installationCode}", installationCode);
+                return NotFound("Echo mission not found");
+            }
+
+            _logger.LogError(e, "Error getting mission from Echo");
+            return new StatusCodeResult(StatusCodes.Status502BadGateway);
+        }
+        catch (JsonException e)
+        {
+            _logger.LogError(e, "Error deserializing mission from Echo");
+            return new StatusCodeResult(StatusCodes.Status500InternalServerError);
+        }
+    }
+
+    [HttpGet]
+    [Route("echo-plant-info")]
+    [ProducesResponseType(typeof(List<EchoPlantInfo>), StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status400BadRequest)]
+    [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+    [ProducesResponseType(StatusCodes.Status403Forbidden)]
+    [ProducesResponseType(StatusCodes.Status404NotFound)]
+    [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+    [ProducesResponseType(StatusCodes.Status502BadGateway)]
+    public async Task<ActionResult<EchoMission>> GetEchoPlantInfos()
```

Comment:
We could probably use the fusion context ([docs](https://docs.fusion-dev.net/)) instead of getting this from echo, but if we do get it from echo, it should be its own controller since it has nothing to do with EchoMissions.

Reply:
It could probably be collected from fusion. This use one of the API endpoints that the Echo Mission planner use to get plant info. I propose that we continue to use this for this PR and can have an issue to consider collecting this from fusion instead.

Reply:
Ok, then it should be its own controller "EchoPlantInfo"